### ISSUE TITLE
Ubuntu 18.04 AutoBuild eosio_build_ubuntu.sh(7 line)  CPU_CORE is empty

### ISSUE THREAD

**Issue by biezhihua** [EOSIO/eos#4920]
HI,developer:

I try use the `eosio_build.sh` script to autobuild eosio, but build error.

I found the script failed to get the correct value（`CPU_CORE`）

```
CPU_CORE=$( lscpu | grep "^CPU(s)" | tr -s ' ' | cut -d\  -f2 || cut -d' ' -f2 )
```

My computer config : 
```
➜  ~ lscpu              
架构：           x86_64
CPU 运行模式：   32-bit, 64-bit
字节序：         Little Endian
CPU:             4
在线 CPU 列表：  0-3
每个核的线程数： 2
每个座的核数：   2
座：             1
NUMA 节点：      1
厂商 ID：        GenuineIntel
CPU 系列：       6
型号：           42
型号名称：       Intel(R) Core(TM) i5-2450M CPU @ 2.50GHz
步进：           7
CPU MHz：        2893.472
CPU 最大 MHz：   3100.0000
CPU 最小 MHz：   800.0000
BogoMIPS：       4988.74
虚拟化：         VT-x
L1d 缓存：       32K
L1i 缓存：       32K
L2 缓存：        256K
L3 缓存：        3072K
NUMA 节点0 CPU： 0-3
标记：           fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx rdtscp lm constant_tsc arch_perfmon pebs bts rep_good nopl xtopology nonstop_tsc cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic popcnt tsc_deadline_timer aes xsave avx lahf_lm epb pti ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid xsaveopt dtherm ida arat pln pts
➜  ~ 

```

**Comment by lazaridiscom** [comment]
@jgiszczak , to avoid changing behavior on other systems, possibly the existent line

`CPU_CORE=$( lscpu | grep "^CPU(s)" | tr -s ' ' | cut -d\  -f2 || cut -d' ' -f2 )`

should stay as-is.

if CPU_CORE is empty after this (like on the machine of @biezhihua ), then add an additional check (possibly based on `lscpu -pCPU') could be done.

**Comment by biezhihua** [comment]
OK. I will paste the output of `lscpu -pCPU`  today.

**Comment by lazaridiscom** [comment]
More re the `-J` option mentioned by heifner above: https://github.com/EOSIO/eos/pull/4870#issuecomment-409434960

**Comment by biezhihua** [comment]
@jgiszczak  

```
➜  ~ lscpu -pCPU
# 以下是可解析格式，可以输给其他程序。每列中的各个不同项有一个惟一
# 的从 0 开始的 ID。
# CPU
0
1
2
3
➜  ~ 
➜  ~ 

```

**Comment by jgiszczak** [comment]
Fix will be in v1.1.2.
